integration_tests/typescript-coverage/typescript-preprocessor.js

```diff
@@ -0,0 +1,20 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+const tsc = require('typescript');
+
+module.exports = {
+  process(src, path) {
```

Comment:
we should make typescript-jest and support it exactly like we support babel. This makes integration easier for people.

I already own the name on npm.